app/domain/logs.rb

```diff
@@ -591,106 +591,106 @@ module AuthnJwt
+        msg:  "Claims denylist value is '{0-deny-claims-list}'",
         code: "CONJ00117D"
       )
 
       ValidatingClaimName = ::Util::TrackableLogMessageClass.new(
-        msg: "Validating claim name '{0-claim-name}'",
+        msg:  "Validating claim name '{0-claim-name}'",
         code: "CONJ00118D"
       )
 
       ValidatedClaimName = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully validated claim name '{0-claim-name}'",
+        msg:  "Successfully validated claim name '{0-claim-name}'",
         code: "CONJ00119D"
       )
       ParsingMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Parsing mandatory claims value '{0-mandatory-claims}'",
+        msg:  "Parsing mandatory claims value '{0-mandatory-claims}'",
         code: "CONJ00120D"
       )
 
       ParsedMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully parsed mandatory claims '{0-mandatory-claims-list}'",
+        msg:  "Successfully parsed mandatory claims '{0-mandatory-claims-list}'",
         code: "CONJ00121D"
       )
 
       FetchingMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Fetching mandatory claims...",
+        msg:  "Fetching mandatory claims...",
         code: "CONJ00122D"
       )
 
       NotConfiguredMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Mandatory claims is not configured",
+        msg:  "Mandatory claims is not configured",
         code: "CONJ00123D"
       )
 
       FetchedMandatoryClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully fetched mandatory claims '{0-mandatory-claims}'",
+        msg:  "Successfully fetched mandatory claims '{0-mandatory-claims}'",
         code: "CONJ00124I"
       )
 
       ParsingMappingClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Parsing mapping claims value '{0-mapping-claims}'",
+        msg:  "Parsing mapping claims value '{0-mapping-claims}'",
         code: "CONJ00125D"
       )
 
       ParsedMappingClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully parsed mapping claims '{0-mapping-claims-table}'",
+        msg:  "Successfully parsed mapping claims '{0-mapping-claims-table}'",
         code: "CONJ00126D"
       )
 
       ClaimMapDefinition = ::Util::TrackableLogMessageClass.new(
-        msg: "Mapping annotation name '{0-annotation-value}' to the claim name '{1-claim-name}'",
+        msg:  "Mapping annotation name '{0-annotation-value}' to the claim name '{1-claim-name}'",
         code: "CONJ00127D"
       )
 
       FetchingMappingClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Fetching mapping claims...",
+        msg:  "Fetching mapping claims...",
         code: "CONJ00128D"
       )
 
       NotConfiguredMappingClaims = ::Util::TrackableLogMessageClass.new(
-        msg: "Mapping claims are not configured",
+        msg:  "Mapping claims are not configured",
```

Comment:
```suggestion
        msg:  "No mapping claims configured",
```

Reply:
It means that `mapping-claims` variable is not defined in authenticator's policy
Please re-review the message